servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/Executors.java

```diff
@@ -48,6 +49,22 @@ public static Executor immediate() {
         return IMMEDIATE_EXECUTOR;
     }
 
+    /**
+     * Returns a global {@link Executor} instance, which should be used for operations that do not involve I/O and
+     * are expected to happen concurrently.
+     * It creates as many threads as required but reuses threads when possible. It is therefore 'safe to block' when
+     * using it.
+     * <p>
+     * <strong>Be cautious</strong> though, and prefer creating dedicated {@link Executor executors}
+     * with proper limits that match the profile of your application and use case
```

Comment:
Consider removing this suggestion for now. Executors with limits (queues) may lead to a deadlock if one callback depends on another one queued for later execution.